components/TextFields/src/MDCTextInputControllerBase.h

```diff
@@ -71,6 +71,13 @@ extern const CGFloat MDCTextInputControllerBaseDefaultBorderRadius;
  */
 @property(class, nonatomic, null_resettable, strong) UIColor *borderFillColorDefault;
 
+/**
+ The color the input field's border in the resting state. A nil value yields a clear border.
```

Comment:
A nil value uses the normal color.

at least that is my read of the code.

Reply:
The base class is shared with both Filled and outlined buttons, so the default is a clear border.